lib/FilesError.js

```diff
@@ -40,7 +40,7 @@ function logAndThrow (e) {
 E('ERROR_INTERNAL', '%s')
 E('ERROR_BAD_ARGUMENT', '%s')
 E('ERROR_NOT_IMPLEMENTED', 'method `%s` not implemented')
-E('ERROR_BAD_CREDENTIALS', 'cannot access `%s`, make sure your credentials are valid')
+E('ERROR_BAD_CREDENTIALS', 'unauthorized access to the %s, please make sure that you are using valid credentials and that your action is executed from Runtime')
```

Comment:
"from Adobe I/O Runtime" rather than just Runtime?

Reply:
ha had the same thought already done :)